recipe/meta.yaml

```diff
@@ -21,28 +21,31 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - julia
+    - jupyter_contrib_core
   run:
-    - sos-notebook >=0.19.4
+    - sos-notebook >=0.22.7
     - feather-format
     - julia
-    - python >=3.6
+    - python
 
 test:
   requires:
     - nose
     - pytest
     - selenium
+    - jupyter_contrib_core
```

Comment:
Does it really need `jupyter_contrib_core`? That hasn't been updated in 4 years...

Reply:
I do not know but the error message said something like missing this one when running the test